Assets/HoloToolkit/Utilities/Editor/AutoConfigureMenu.cs

```diff
@@ -0,0 +1,182 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Configuration options derived from here: 
+    /// https://developer.microsoft.com/en-us/windows/holographic/unity_development_overview#Configuring_a_Unity_project_for_HoloLens
+    /// </summary>
+    public static class AutoConfigureMenu
+    {
+        /// <summary>
+        /// Applies recommended scene settings to the current scenes
+        /// </summary>
+        [MenuItem("HoloToolkit/Auto Configure/Scene", false)]
```

Comment:
Someone looking at this menu might not quite know what it is for. Suggestion: "HoloToolkit/Configure/Apply HoloLens Settings to Scene". Too long? Just an idea.